tests/test_timereport.py

```diff
@@ -25,15 +25,22 @@ def test_slack_payload_extractor_message():
     assert fake_data.get('text') == ['fake text']
 
 
-def test_factory():
-    fake_order = dict(user_id='fake', user_name='fake mcFake', text=['fake_cmd=do_fake fake_reason 2018-01-01'])
+def fake_factory(order):
```

Comment:
Instead of doing it this way you could leverage pytest parameterized:

```
@pytest.mark.parametrize("test_input", [
    ("foo"),
    ("bar"),
])
test_foo_or_bar(test_input):
  assert your_function()...
```

https://docs.pytest.org/en/latest/parametrize.html